templates/common/_base/units/machine-config-daemon-pull.service.yaml

```diff
@@ -0,0 +1,28 @@
+name: "machine-config-daemon-pull.service"
+enabled: true
+contents: |
+  [Unit]
+  Description=Machine Config Daemon Pull
+  # Make sure it runs only on OSTree booted system
+  ConditionPathExists=/run/ostree-booted
+  BindsTo=ignition-firstboot-complete.service
```

Comment:
This was just removed in https://github.com/openshift/machine-config-operator/pull/1762